src/PVRIptvData.cpp

```diff
@@ -274,11 +274,24 @@ bool PVRIptvData::LoadEPG(time_t iStart, time_t iEnd)
     entry.iGenreType = 0;
     entry.iGenreSubType = 0;
     entry.strPlotOutline = "";
+    entry.strOriginalTitle = "";
+    entry.strCast = "";
+    entry.strDirector = "";
+    entry.strWriter = "";
+    entry.iYear = 0;
+    entry.strIMDBNumber = "";
+    entry.strEpisodeName = "";
     entry.startTime = iTmpStart;
     entry.endTime = iTmpEnd;
 
     GetNodeValue(pChannelNode, "title", entry.strTitle);
     GetNodeValue(pChannelNode, "desc", entry.strPlot);
+    GetNodeValue(pChannelNode, "sub-title", entry.strOriginalTitle);
```

Comment:
This should read `entry.episodeName`.

Reply:
That’s incorrect, sub-title Is for any language including English. @ksooo is right it should be set to Episode name.

Reply:
```
  <programme start="20190616053000 +0200" stop="20190616053000 +0200" channel="EDGEsport">
    <title lang="en">Live Recap Asia Talent Cup 2018</title>
    <sub-title lang="en">Malaysia - Race 2</sub-title>
    <desc lang="en">Live Recap. Relive the action from the Asia Talent Cup. The competition sees Asia's stars of the future participate in a ten race championship held over five rounds throughout the year, with all of the riders on homogenised Honda NSF250R Moto3™ bikes(n)</desc>
  </programme>
```

In the above sub-title is the episode name.